### PR TITLE
Test ast

### PR DESCRIPTION
**Pull request by pbrunet**

### CODE REVIEW

File: src/libtriton/CMakeLists.txt [hunk 1]
```diff
@@ -94,7 +94,6 @@ set(LIBTRITON_SOURCE_FILES
     format/pe/peOptionalHeader.cpp
     format/pe/peSectionHeader.cpp
     modes/modes.cpp
-    os/unix/syscallNumberToString.cpp
```
Comment (JonathanSalwan) [inline src/libtriton/CMakeLists.txt]: don't remove this file

File: src/libtriton/ast/z3/z3ToTritonAst.cpp [hunk 2]
```diff
@@ -149,7 +149,7 @@ namespace triton {
           break;
         }
 
-        //case Z3_OP_BSDIV_I:
```
Comment (JonathanSalwan) [inline src/libtriton/ast/z3/z3ToTritonAst.cpp]: keep it commented. it's only available on the next z3 release.

File: src/libtriton/ast/z3/z3ToTritonAst.cpp [hunk 3]
```diff
@@ -149,7 +149,7 @@ namespace triton {
...
+        case Z3_OP_BSDIV_I:
```
Comment (JonathanSalwan) [inline src/libtriton/ast/z3/z3ToTritonAst.cpp]: keep it commented. it's only available on the next z3 release.

File: src/libtriton/ast/z3/z3ToTritonAst.cpp [hunk 4]
```diff
@@ -159,7 +159,7 @@ namespace triton {
           break;
         }
 
-        //case Z3_OP_BUDIV_I:
```
Comment (JonathanSalwan) [inline src/libtriton/ast/z3/z3ToTritonAst.cpp]: keep it commented. it's only available on the next z3 release.

File: src/libtriton/ast/z3/z3ToTritonAst.cpp [hunk 5]
```diff
@@ -169,7 +169,7 @@ namespace triton {
           break;
         }
 
-        //case Z3_OP_BSREM_I:
```
Comment (JonathanSalwan) [inline src/libtriton/ast/z3/z3ToTritonAst.cpp]: keep it commented. it's only available on the next z3 release.

File: src/libtriton/ast/z3/z3ToTritonAst.cpp [hunk 6]
```diff
@@ -179,7 +179,7 @@ namespace triton {
           break;
         }
 
-        //case Z3_OP_BUREM_I:
```
Comment (JonathanSalwan) [inline src/libtriton/ast/z3/z3ToTritonAst.cpp]: keep it commented. it's only available on the next z3 release.

File: src/libtriton/ast/z3/z3ToTritonAst.cpp [hunk 7]
```diff
@@ -189,7 +189,7 @@ namespace triton {
           break;
         }
 
-        //case Z3_OP_BSMOD_I:
```
Comment (JonathanSalwan) [inline src/libtriton/ast/z3/z3ToTritonAst.cpp]: keep it commented. it's only available on the next z3 release.

File: src/libtriton/bindings/python/objects/pyAstNode.cpp [hunk 8]
```diff
@@ -577,8 +565,8 @@ namespace triton {
         AstNode_operatorSub,                        /* nb_subtract */
         AstNode_operatorMul,                        /* nb_multiply */
         AstNode_operatorDiv,                        /* nb_divide */
-        AstNode_operatorRem,                        /* nb_remainder */
-        AstNode_operatorMod,                        /* nb_divmod */
+        AstNode_operatorMod,                        /* nb_remainder */
```
Comment (JonathanSalwan) [inline src/libtriton/bindings/python/objects/pyAstNode.cpp]: AstNode_operatorMod -> AstNode_operatorRem

File: src/libtriton/bindings/python/objects/pyAstNode.cpp [hunk 9]
```diff
@@ -600,7 +588,7 @@ namespace triton {
         AstNode_operatorSub,                        /* nb_inplace_subtract */
         AstNode_operatorMul,                        /* nb_inplace_multiply */
         AstNode_operatorDiv,                        /* nb_inplace_divide */
-        AstNode_operatorRem,                        /* nb_inplace_remainder */
```
Comment (JonathanSalwan) [inline src/libtriton/bindings/python/objects/pyAstNode.cpp]: AstNode_operatorMod -> AstNode_operatorRem

File: src/libtriton/includes/triton/unix.hpp [hunk 10]
```diff
@@ -1,56 +0,0 @@
-//! \file
```
Comment (JonathanSalwan) [inline src/libtriton/includes/triton/unix.hpp]: Don't remove this file.

File: src/libtriton/os/unix/syscallNumberToString.cpp [hunk 11]
```diff
@@ -1,40 +0,0 @@
-//! \file
```
Comment (JonathanSalwan) [inline src/libtriton/os/unix/syscallNumberToString.cpp]: Don't remove this file.

File: src/libtriton/ast/z3/z3ToTritonAst.cpp (hunk 3 above)
Comment (pbrunet) [inline src/libtriton/ast/z3/z3ToTritonAst.cpp]: It makes triton crash on my computer if I comment it and use operators

File: src/libtriton/os/unix/syscallNumberToString.cpp (hunk 11 above)
Comment (pbrunet) [inline src/libtriton/os/unix/syscallNumberToString.cpp]: why do we need it? It is nor use, nor tested (nor documented)

Comment (JonathanSalwan) [inline src/libtriton/os/unix/syscallNumberToString.cpp]: i know. i still have to add the method.

File: src/libtriton/ast/z3/z3ToTritonAst.cpp (hunk 3 above)
Comment (JonathanSalwan) [inline src/libtriton/ast/z3/z3ToTritonAst.cpp]: maybe but it must working on the last z3 release. can you provide a poc because it works on my computer.

Comment (pbrunet) [inline src/libtriton/ast/z3/z3ToTritonAst.cpp]: I am working with a z3 release (4.5.0) so it is the last one.

**Comment by pbrunet** [comment]
Sadly, this MR show that z3 4.4.1 is not reliable enough on simplification :s (osx which use 4.5 works better)

**Comment by pbrunet** [comment]
This one is OK, You just need to rerun the 404 error

**Pull request merged**